theano/tensor/subtensor.py

```diff
@@ -1943,8 +1941,7 @@ def copy_of_x(self, x):
                 NPY_ARRAY_ENSURECOPY, NULL)""" % locals()
 
     def c_support_code(self):
-        from theano.gof.cutils import compile_cutils_code
-        return compile_cutils_code()
+        return inc_code()
```

Comment:
The code is compiled exactly as often as before for the op.

I just removed the copy in cutils_ext.